lms/migrations/versions/396f46318023_events_backfill.py

```diff
@@ -27,9 +34,7 @@ def get_configured_launch_type_pk(conn):
     ).fetchone()[0]
 
 
-def upgrade():
-    conn = op.get_bind()
-
+def migrate_lti_launches(conn, start, end):
```

Comment:
Making these methods to import them in the next batches

Reply:
Can you import directly, or will we be copy pasting?

Reply:
Not sure, I was thinking importing but we'd have to rename this migration file to be a valid python name.

Alembic doesn't care about the file names so we could do that on the second PR after this.